grading/ipmininet/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG   VERSION=latest
+FROM  ingi/inginious-c-base:${VERSION}
+LABEL org.inginious.grading.name="ipmininet"
+
+# Install mininet deps
+RUN	dnf install -y git make gcc centos-release-nfv-openvswitch &&\
+	dnf install -y openvswitch3.1 &&\
+	pip3 install pyflakes
+
+# Install mininet
+WORKDIR /opt
+RUN 	git clone https://github.com/mininet/mininet &&\
+	git -C mininet checkout 2.3.1b1 &&\
+	ln -s /bin/python3 /bin/python &&\
```

Comment:
Is this safe on EL8 ?

Reply:
I did not test yet this container as a grading environment.
That being said, I'm not aware of any INGInious parts still relying on python2.
This is a quick fix to make the `mn` command functionnal within the container.

Reply:
This does not seem to be an issue in my test instance.